### PR TITLE
TELCODOCS-1191: Clean up of SNO references in TALM docs

### PR DESCRIPTION
**Pull request by amolnar-gh**
Version(s): 4.12, 4.13

Issue: https://issues.redhat.com/browse/TELCODOCS-1191

Link to docs preview:
* https://56826--docspreview.netlify.app/openshift-enterprise/latest/scalability_and_performance/cnf-talm-for-cluster-upgrades.html#cnf-about-topology-aware-lifecycle-manager-config_cnf-topology-aware-lifecycle-manager
* https://56826--docspreview.netlify.app/openshift-enterprise/latest/scalability_and_performance/cnf-talm-for-cluster-upgrades.html#creating_backup_cnf-topology-aware-lifecycle-manager
* https://56826--docspreview.netlify.app/openshift-enterprise/latest/scalability_and_performance/cnf-talm-for-cluster-upgrades.html#talo-backup-start_and_update_cnf-topology-aware-lifecycle-manager
* https://56826--docspreview.netlify.app/openshift-enterprise/latest/scalability_and_performance/cnf-talm-for-cluster-upgrades.html#talo-precache-feature-concept_cnf-topology-aware-lifecycle-manager

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/cnf-about-topology-aware-lifecycle-manager-config.adoc [hunk 1]
```diff
@@ -16,6 +16,9 @@ The {cgu-operator-first} manages the deployment of {rh-rhacm-first} policies for
 * The order of policies remediated to the cluster
 * The assignment of a canary cluster
 
-For {sno}, the {cgu-operator-first} can create a backup of a deployment before an upgrade. If the upgrade fails, you can recover the previous version and restore a cluster to a working state without requiring a reprovision of applications.
+For {sno}, the {cgu-operator-first} offers the following features:
+
+* create a backup of a deployment before an upgrade
+* pre-caching images for clusters with limited bandwidth
```
Comment (adellape) [inline modules/cnf-about-topology-aware-lifecycle-manager-config.adoc:22]: Capitalize first letter in each list entry ([ISG](https://www.ibm.com/docs/en/ibm-style?topic=format-lists#capitalization-in-lists))

File: modules/cnf-topology-aware-lifecycle-manager-precache-feature.adoc
Comment (sudomakeinstall2) [inline modules/cnf-topology-aware-lifecycle-manager-precache-feature.adoc:1]: line 114:
oc get jobs,pods -n openshift-talm-pre-cache
should be changed to
oc get jobs,pods -n openshift-talo-pre-cache

Comment (amolnar-rh) [reply modules/cnf-topology-aware-lifecycle-manager-precache-feature.adoc:1]: Thank you, corrected!

**Comment by amolnar-rh** [comment]
Please, hold off on merging for now

**Pull request merged**